src/stores/game.js

```diff
@@ -365,7 +365,24 @@ export const useGameStore = defineStore('game', {
           return reject(jwres.body.message);
       }
     },
-
+    makeSocketRequest(slug, body, method = 'get') {
+      const url =
+        import.meta.env.VITE_USE_GAMESTATE_API === 'true'
+          ? `/api/gamestate/${this.id}/move/${slug}`
+          : `/api/game/${slug}`;
+      return new Promise((resolve, reject) => {
+        io.socket.request(
+          {
+            method: method,
+            url: url,
+            data: body,
```

Comment:
Let's use the shorthand here. For the same reason let's rename body to data
```suggestion
            method,
            url,
            data,
```